object-manipulation-1/src/get-value.js

```diff
@@ -0,0 +1,8 @@
+/* exported getValue */
+
+/* return object[key] to get the value of the property */
```

Comment:
notice that your pseudocode is not including the step of saving the value in the variable, and you don't have a step which states to return the value. make sure the pseudo code represents all of the steps being completed.